readme/build_troubleshooting.md

```diff
@@ -21,6 +21,13 @@ If you get a node-gyp related error, you might need to manually install it: `npm
 
 If you get the error `libtool: unrecognized option '-static'`, follow the instructions [in this post](https://stackoverflow.com/a/38552393/561309) to use the correct libtool version.
 
+If you there is an error `/joplin/ReactNativeClient/ios/Pods/Target Support Files/Pods-Joplin/Pods-Joplin.debug.xcconfig: unable to open file (in target "Joplin" in project "Joplin") (in target 'Joplin' from project 'Joplin')` run following commands:
+
+    cd ios
+    pod deintegrate
+    pod install
+
+
```

Comment:
Please create a new section `# Mobile application`, then a sub-section `## iOS` and put your text there.

Reply:
> Please create a new section `# Mobile application`, then a sub-section `## iOS` and put your text there.
Done. Please review my changes.

Reply:
I wrote `# Mobile application` and `## iOS` and you've added `#Mobile application` and `##IOS`, which is not even valid Markdown. I wrote "iOS" and you wrote "IOS". Details are important in software development.

Reply:
Sorry about that. WebStorm renders ##iOS without any issues, must be some mistake